assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/daemonset.yaml

```diff
@@ -133,13 +133,15 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            # This env var enables calico to expose prometheus metrics.
```

Comment:
Is this something Lokomotive specific that we've added ? Perhaps we should a  standard uniform comment tag which can represent the downstream changes done for Lokomotive.

Reply:
@ipochi what do you mean here? I don't fully understand what needs to be done?

Reply:
I merely pointed out that if any Lokomotive specific changes are done to the yamls then we should atleast have some uniformity in the comments.

something like `# Lokomotive Specific changes, do not delete during upgrade` or something smaller that could be part of every change in the yaml, so that accidently the changes are not deleted.

Reply:
Historically we have used something like this: 

https://github.com/kinvolk/lokomotive/blob/68df4344cd9abae7ca1c2a6a69f129b6f3e0ffa0/assets/components/contour/contour/03-envoy.yaml#L2-L4 making such changes then.

Reply:
@ipochi done, please check.